x-pack/plugins/apm/server/lib/observability_overview/get_service_count.ts

```diff
@@ -6,47 +6,35 @@
 
 import { ProcessorEvent } from '../../../common/processor_event';
 import { rangeFilter } from '../../../common/utils/range_filter';
-import {
-  SERVICE_NAME,
-  PROCESSOR_EVENT,
-} from '../../../common/elasticsearch_fieldnames';
+import { SERVICE_NAME } from '../../../common/elasticsearch_fieldnames';
 import { Setup, SetupTimeRange } from '../helpers/setup_request';
 
 export async function getServiceCount({
   setup,
 }: {
   setup: Setup & SetupTimeRange;
 }) {
-  const { client, indices, start, end } = setup;
+  const { apmEventClient, start, end } = setup;
 
   const params = {
-    index: [
-      indices['apm_oss.transactionIndices'],
-      indices['apm_oss.errorIndices'],
-      indices['apm_oss.metricsIndices'],
-    ],
+    apm: {
+      events: [
+        ProcessorEvent.transaction,
```

Comment:
We use this array in a few places. Maybe there's a good place to export a `uiProcessorEvents` that we can reuse.

Reply:
UIProcessorEvent is specifically meant for `public` (the query bar). I don't think it belongs in `server` code.